deploy/base/easi/easi-frontend_deployment.yaml

```diff
@@ -23,4 +27,8 @@ spec:
             - containerPort: 80
               protocol: TCP
           resources: {}
+          volumeMounts:
+          - name: nginx-config
+            mountPath: /etc/nginx/conf.d/easi.conf
+            subPath: easi.conf
```

Comment:
Does `nginx` just take any `.conf` file in `/etc/nginx/conf.d/`? Just wanted to make sure I understood how we could get away with this `easi.conf` name without presumably configuring nginx to look for it 👀

Reply:
Correct - the default.conf has an INCLUDE line that includes anything in /etc/nginx/conf.d.